package.json

```diff
@@ -3,31 +3,49 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./scripts/www",
-    "debug": "cross-env DEBUG=bbsite nodemon ./scripts/www",
-    "lint": "eslint --ext .js --ext .jsx src test templates scripts app.js gulpfile.js",
-    "test": "npm run lint && gulp",
+    "copy-client-scripts": "./scripts/copy-client-scripts.sh",
+    "mkdirs": "mkdir -p static/js/editor static/js/entity static/stylesheets",
+    "clean": "./scripts/clean.sh",
+    "prepublish": "npm run clean && npm run mkdirs && npm run copy-client-scripts",
+    "build-server-js": "babel src --out-dir lib",
```

Comment:
It seems like this creates an implicit, undocumented dep on a global babel install (which is also currently not reflected in the travis config). Could we depend on a local babel install for this?